bedrock/firefox/templates/firefox/welcome/page8.html

```diff
@@ -64,7 +64,7 @@
         heading_level=1
       ) %}
       <p class="primary-cta protection-report">
-        <a href="https://support.mozilla.org/kb/enhanced-tracking-protection-firefox-desktop/?utm_source={{ _utm_source }}&utm_medium=referral&utm_campaign={{ _utm_campaign }}&utm_content={{_utm_content}}" class="mzp-c-button mzp-t-product" id="button" rel="external">
+        <a href="https://support.mozilla.org/kb/enhanced-tracking-protection-firefox-desktop/?utm_source={{ _utm_source }}&utm_medium=referral&utm_campaign={{ _utm_campaign }}&utm_content={{_utm_content}}" class="mzp-c-button mzp-t-product" id="button" rel="external" data-cta-text="View your report - no JS" data-cta-type="button">
```

Comment:
Since these attributes are evidently required even when there's an explicit event, I don't think we should say "no JS" in the name.

Reply:
We should also add `data-cta-position="Primary"` to help further distinguish the big button from the secondary link below.